2-Browsers/Week1/homework/ex5-catWalk/index.js

```diff
@@ -21,8 +21,39 @@ Full description at: https://github.com/HackYourFuture/Homework/tree/main/2-Brow
 
    https://media1.tenor.com/images/2de63e950fb254920054f9bd081e8157/tenor.gif
 -----------------------------------------------------------------------------*/
+let dancing = false;
 function catWalk() {
-  // TODO complete this function
+  function roundToNearestTen(number) {
+    return Math.floor(number / 10) * 10; // because the width of the screen is
+  }
+  const imgRef = document.querySelector('img');
```

Comment:
Again fantastic work! 💪 Only thing I would recommend is to leave some more space between the lines of code. It is easier to read if there is some space between. For example, if you have a new line between this function declaration and the variable.

```suggestion
  function roundToNearestTen(number) {
    return Math.floor(number / 10) * 10; // because the width of the screen is
  }
  
  const imgRef = document.querySelector('img');
```